spec/models/external_training_spec.rb

```diff
@@ -8,7 +8,7 @@
 require 'spec_helper'
 
 describe ExternalTraining do
-  let(:today) { Date.new(2024, 3, 26) }
+  let(:today) { Date.current }
```

Comment:
Working with `Date.current` may lead to flaky specs. Not in this case I think, but generally. Consider using https://github.com/travisjeffery/timecop to freeze the current time, so `Date.current` returns the same for the whole execution of the spec.